packages/account/src/Components/icon-message-list/list-item.tsx

```diff
@@ -2,17 +2,17 @@ import React from 'react';
 import { Text } from '@deriv/components';
 import { observer, useStore } from '@deriv/stores';
 import { Localize } from '@deriv/translations';
+import { useDevice } from '@deriv-com/ui';
 
 type TListItem = {
     text: React.ReactNode;
     index?: number;
 };
 
 const ListItem = observer(({ text, index }: TListItem) => {
```

Comment:
Hi @Dhruv-deriv ,

Now as you have already removed `store` so you can remove this `observer` as well .

@amina-deriv please correct me if I am wrong thank you 🙇

Reply:
hey @fasihali-deriv  we are using observer here should we still remove it ?

<img width="670" alt="Screenshot 2024-06-04 at 2 49 12 PM" src="https://github.com/fasihali-deriv/deriv-app/assets/158162395/0d891860-549d-4dd4-bdfa-691eb56b78e0">

Reply:
hey @fasihali-deriv i have removed it as we dont need observer thanks 👍